src/map/battle.cpp

```diff
@@ -3903,6 +3905,16 @@ static void battle_calc_attack_masteries(struct Damage* wd, struct block_list *s
 #endif
 				}
 				break;
+#ifdef RENEWAL
+			case GN_CARTCANNON:
+				// These skills add arrow attack as mastery attack instead of equip attack
+				// The only way to determine this is by confirming that the arrow attack isn't influenced by P.ATK
```

Comment:
Is this a documentation of how you found out that it should be changed like this or a TODO for the future?

Reply:
Both - so whenever someone wonders whether to add a skill to the list here or not they know how they can verify that.

Reply:
Rephrased so it reflects reality better.